src/components/common/ColonyActions/ActionsListItem/helpers.ts

```diff
@@ -1,64 +1,53 @@
 import { useGetMotionStateQuery } from '~gql';
 import { useColonyContext, useEnabledExtensions } from '~hooks';
 import { motionTags } from '~shared/Tag';
-import { MotionData } from '~types';
-import {
-  MotionState,
-  getMotionState,
-  shouldDisplayMotionCountdownTime,
-} from '~utils/colonyMotions';
+import { Address, MotionData } from '~types';
+import { MotionState, getMotionState } from '~utils/colonyMotions';
 
-const getMotionTag = (
-  isMotion: boolean | null | undefined,
-  motionState: MotionState,
-) => {
-  // If not motion, then forced action
-  if (!isMotion) {
-    return motionTags.Forced;
-  }
-
-  return motionTags[motionState];
-  // @TODO: handle case where motion is from now-uninstalled voting rep version.
-};
-
-export const useMotionStatusDisplay = (
+export const useColonyMotionState = (
   isMotion: boolean | null | undefined,
   motionData: MotionData | null | undefined,
+  transactionHash: Address,
 ) => {
-  const { isVotingReputationEnabled } = useEnabledExtensions();
   const { colony } = useColonyContext();
+  const { isVotingReputationEnabled } = useEnabledExtensions();
 
-  const {
-    data: motionStateData,
-    loading: loadingMotionState,
-    refetch: refetchMotionState,
-  } = useGetMotionStateQuery({
-    skip: !isMotion || !motionData || !colony,
-    variables: {
-      input: {
-        colonyAddress: colony?.colonyAddress ?? '',
-        motionId: Number(motionData?.motionId),
+  const { data: motionStateData, refetch: refetchMotionState } =
+    useGetMotionStateQuery({
+      skip: !isMotion || !motionData || !colony || !isVotingReputationEnabled,
+      variables: {
+        input: {
+          colonyAddress: colony?.colonyAddress ?? '',
+          transactionHash,
+        },
       },
-    },
-  });
+      fetchPolicy: 'cache-and-network',
+    });
   const networkMotionState = motionStateData?.getMotionState;
 
-  const showMotionTag = !loadingMotionState && isVotingReputationEnabled;
```

Comment:
Since I've changed the fetch policy, hiding the tag if its loading essentially renders the cache ineffective since it'll hide the tag even if we have cached data available.